frontend/App.jsx

```diff
@@ -0,0 +1,28 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
```

Comment:
Cool that you're using axios. I haven't heard about that before - so you'll have to tell me all about it later! It's used instead of the usual fetch right?